Platon/backend/app/auth_system/models.py

```diff
@@ -12,10 +12,11 @@ class User(db.Model):
     surname = db.Column(db.String(50),nullable=False)
     is_private = db.Column(db.Boolean,nullable=False)
     profile_photo = db.Column(db.String(50))
-    google_scholar_name = db.Column(db.String(50))
-    researchgate_name = db.Column(db.String(50))
+    google_scholar_name = db.Column(db.String(256))
```

Comment:
@HUmutOzdemir

I have a critical note for this small update:

Alembic does not detect such "small" updates on schema. So, unfortunately, the "User" model in the database does not get modified even after the migrations are run.

So, to reconcile this modification in the database, we need to update the schema **MANUALLY**.

Here's what I ask from you step by step:
1. Run "sudo docker-compose up --build" to start our services
2. Open another shell. Run " sudo docker exec -it platon_db_1 /bin/bash -l".
3. After you have connected to the database container, run "mysql -uuser -ppassword".
4. After you connect to the MySQL server, run these commands respectively: "use platondb;", "ALTER TABLE users MODIFY COLUMN google_scholar_name VARCHAR(256);", "ALTER TABLE users MODIFY COLUMN researchgate_name VARCHAR(256);".
5. Exit.

Reply:
@HUmutOzdemir 

The reason why I needed to change the column length was that 50 characters were not long enough to store URLs. So, I had to make this change on schema.

Reply:
I will change it when I deployed this code on the server. Thank you for your good explention.